packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.editorConfig.ts

```diff
@@ -35,32 +35,36 @@ export const getPreview = (values: DatagridDateFilterPreviewProps): StructurePre
                 columnSize: "grow",
                 backgroundColor: "#FFFFFF",
                 children: [
-                    {
-                        type: "Container",
-                        padding: 4,
-                        grow: 0,
-                        children: [
-                            {
-                                type: "Image",
-                                document: getSvgContent(values.defaultFilter)
-                            } as ImageProps
-                        ]
-                    } as ContainerProps,
-                    {
-                        type: "Container",
-                        borders: true,
-                        borderWidth: 0.5,
-                        grow: 0
-                    } as ContainerProps,
+                    ...(values.adjustable
+                        ? [
+                              {
+                                  type: "Container",
+                                  padding: 2,
+                                  grow: 0,
+                                  children: [
+                                      {
+                                          type: "Image",
+                                          document: getSvgContent(values.defaultFilter)
+                                      } as ImageProps
+                                  ]
+                              } as ContainerProps,
+                              {
+                                  type: "Container",
+                                  borders: true,
+                                  borderWidth: 0.5,
+                                  grow: 0
+                              } as ContainerProps
+                          ]
```

Comment:
After going through all the syntax I understood this, but at first glance it's not quite readable.
I would suggest extracting the entire ternary to a const called something like `adjustableOptionContainers` and then just do `...adjustableOptionContainers` here. I'm not sure about the naming of the const, but more specifically I'm referring to making it more readable through a const

Reply:
Fixed